config/vufind/config.ini

```diff
@@ -120,6 +120,8 @@ offcanvas = false
 showBookBag = false
 ; Set the maximum amount of items allowed in the Book Bag - Default is 100
 bookBagMaxSize = 100
+; Show individual add/remove bookbag buttons in search results
+bookbagTogglesInSearch = 1
```

Comment:
As far as I can see, the only disadvantage to this default is that it makes it much harder to add a whole page of results to the cart. I'm not sure if that is important or not -- but it's the most significant difference I can think of. (I did some testing to confirm that this option works perfectly well with JS disabled, so that's not an issue).